src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.macOS.cs

```diff
@@ -58,7 +58,7 @@ public override void DrawRect(CGRect rect)
 			_drawRect = GetDrawRect(rect);
 			if(UseLayoutManager)
 			{
-				_layoutManager?.DrawBackgroundForGlyphRange(new NSRange(0, (nint)_layoutManager.NumberOfGlyphs), _drawRect.Location);
+				_layoutManager?.DrawGlyphsForGlyphRange(new NSRange(0, (nint)_layoutManager.NumberOfGlyphs), _drawRect.Location);
```

Comment:
I think a comment would be useful here, in case someone should be working on a bug and decide they want to change it back to `DrawBackgroundForGlyphRange()` again...

Reply:
Did you push your updates to explain ? What's the difference between the two ?

Reply:
@jeromelaban just did it.